packages/grant-explorer/src/features/api/rounds.ts

```diff
@@ -103,17 +104,22 @@ query GetRounds(
 }
 `;
 
-export function useRoundsTakingApplications() {
-  const currentTimestamp = Math.floor(Date.now() / 1000).toString();
-  const infiniteTimestamp = ethers.constants.MaxUint256.toString();
+const NOW_IN_SECONDS = Date.now() / 1000;
+const ONE_YEAR = 3600 * 24 * 365;
```

Comment:
would appreciate a SECONDS suffix here to make it super clear what unit this is in.